tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -166,11 +169,20 @@ module.exports = {
      * @returns {string}
      */
     getFileRowSelectorByFileName: function (fileName) {
-      var element = this.elements['fileRowByName']
-      var util = require('util')
+      // replace double quotes if there any to single quotes
       if (fileName.indexOf('"') > -1) {
-        element.selector = element.selector.replace(/"/g, "'")
+        fileName = fileName.replace(/"/g, "'")
+      }
+
+      const parts = path.parse(fileName)
+      if (parts.ext) {
```

Comment:
Currently, we have a bug that shows the link for the folder having a period splitted (ie. without extension). If this gets fixed, we might need to use regex step definition to pass if it's really a folder or a file. This way, we could avoid splitting if it's a folder.